step-release-vis/angular.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
what fw generated this files? was it in some STEP instructions or you found it elsewhere?are we sure we need them?

Reply:
Angular did. Most of the files are necessary, only the app.component.* files in src/app are sample and will be redone.

Reply:
ok, I am fine submitting, but let's add a TODO at the top of all of them to review them later on 1by1 if they are modified to our project's need, or delete them entirely if it turns out we do not  need them.. they are good as examples, but we might be able to optimize some of them out letar on ;)

Reply:
Do you have questions about specific files? Most of them have a comment at the top explaining the purpose. You can also ask me about some of them, I had to do a bit of research to set up the CI.

Reply:
I do get what they do.. I am not sure, having jasmine for unit testing protractor is the best choice for e2e testing, but that highly depends on the structure of the code later as well... it is webdriver based, so likely it is not a super bad choice, but I am not familiar with it yet, so having a TODO to revisit the decision currently implicitly made by the framework once the whole team have seen thee files, and started to figure out what each of you works on still seems to be reasonable for me.

a few of the ideas that came to my mind, so that you can relate:

if we decide to use potractor the hardcoded port, and timeout might be something we want to make more flexible etc, but these we will not know until we start using what is already there. 

the favicon we will likely customize, but that is super low priority

the ts compile cfg we might change to match recent google internal standards in as many flags as possible, if that is something not fully opensourced yet

Reply:
Whoops, didn't notice the favicon, replaced it with Google's logo. Also added TODOs to the configs.

Reply:
TODOs should look the following way:
-exactly one space after the comment marker (good)
-TODO (OK)
-assignment in parenthesis in the form of a username (for sinlge project TODOs, or stuff we do not submit), or bug id
--since you do not have access to the internal issuetracker, both of the above is fine
--github does have a list of issues to be handled at the level of repositories, we can start using that
-what to do

e.g.

// TODO(andreystar): review the autogenerated config, once the file is actually in use, and we have more insight on the real needs